# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1322678752 28800
#      Wed Nov 30 10:45:52 2011 -0800
# Node ID 16b3736e534b130b6780874ebe7d117d919ae822
# Parent  12fa52c6e886cc93e4493a4bb947a8de6ff4ceec
doc: Remove unnecessary reference to deprecated function replot()

help.cc: Remove unnecessary reference to deprecated function replot()

diff --git a/src/help.cc b/src/help.cc
--- a/src/help.cc
+++ b/src/help.cc
@@ -637,23 +637,16 @@ endparfor\n\
 Declare variables as persistent.  A variable that has been declared\n\
 persistent within a function will retain its contents in memory between\n\
 subsequent calls to the same function.  The difference between persistent\n\
 variables and global variables is that persistent variables are local in \n\
 scope to a particular function and are not visible elsewhere.\n\
 @seealso{global}\n\
 @end deftypefn"),
 
-  pair_type ("replot",
-    "-*- texinfo -*-\n\
-@deftypefn {Keyword} {} replot\n\
-Replot a graphic.\n\
-@seealso{plot}\n\
-@end deftypefn"),
-
   pair_type ("return",
     "-*- texinfo -*-\n\
 @deftypefn {Keyword} {} return\n\
 Return from a function.\n\
 @seealso{function}\n\
 @end deftypefn"),
 
   pair_type ("static",
